scripts/test_udp.py

```
@@ -0,0 +1,75 @@
+    # zone_id
+    zone_bytes = zone_id.encode("utf-8")[:31]
+    req[22:22 + len(zone_bytes)] = zone_bytes
+
+    sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    sock.settimeout(2.0)
+
+    print(f"Sending {REQUEST_SIZE}-byte request to {host}:{port}")
+    if zone_id:
+        print(f"  zone_id: {zone_id}")
+    else:
+        print(f"  zone_id: (empty — bridge picks first zone)")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)